.github/workflows/ci-crossdock.yml

```diff
@@ -0,0 +1,67 @@
+name: CIT Crossdock
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  crossdock:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        steps:
+        - name: crossdock
+          cmd: bash scripts/travis/build-crossdock.sh
+        - name: crossdock-otel
+          cmd: make build-crossdock crossdock-otel
+    name: ${{ matrix.steps.name }}
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: true
+
+    - name: Fetch git tags
+      run: |
+        git fetch --prune --unshallow --tags
+
+    - uses: actions/setup-go@v2
+      with:
+        go-version: ^1.15
+
+    - uses: docker/login-action@v1
```

Comment:
we're reusing login and branch logic already, can we move them into reusable actions?

Reply:
I don't think GitHub Actions support calling one action from another action so docker login part isn't possible to move yet but I have moved the logic for populating BRANCH env variable into separate action script.